fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -4258,7 +4259,16 @@ public static void getDdlStmt(String dbName, Table table, List<String> createTab
             }
             // There MUST BE 2 space in front of each column description line
             // sqlalchemy requires this to parse SHOW CREATE TAEBL stmt.
-            sb.append("  ").append(column.toSql());
+            if (table.getType() == TableType.OLAP || table.getType() == TableType.OLAP_EXTERNAL) {
```

Comment:
And also desc statement has the problem, you should hide the aggregate type.

Reply:
the replace aggregate type name will mislead a wrong SQL since that the result of show create table statement is used to execute create table statement,
the desc statement will not be used to do so but merely used to describe the schema of a table exactly.
in other word,
if you want to create a table by the result of show create table statement, do not show the replace aggregate type name since we do not indicate it when execute create table indeed.
if you want to know a table by the result of desc statement, show the replace aggregate type name since it is replace aggregate type indeed.